src/quacc/runners/ase.py

```diff
@@ -309,6 +332,76 @@ def run_vib(self, vib_kwargs: VibKwargs | None = None) -> Vibrations:
 
         return vib
 
+    def run_md(
+        self,
+        timestep: float = 1.0,
+        steps: int = 1000,
+        dynamics: MolecularDynamics = VelocityVerlet,
+        dynamics_kwargs: dict[str, Any] | None = None,
+        initial_temperature: float | None = None,
+        fix_initial_com: bool = True,
+        fix_initial_rot: bool = True,
+        rng_seed: int | None = None,
+    ) -> MolecularDynamics:
+        """
+        Run an ASE-based MD in a scratch directory and copy the results back to
+        the original directory.
+
+        Parameters
+        ----------
+        timestep
+            The time step in femtoseconds.
+        steps
+            Maximum number of steps to run
+        dynamics
+            MolecularDynamics class to use, from `ase.md.md.MolecularDynamics`.
+        dynamics_kwargs
+            Dictionary of kwargs for the dynamics. Takes all valid kwargs for ASE
+            MolecularDynamics classes.
+        initial_temperature
+            If specified, a MaxwellBoltzmannDistribution will be applied to the atoms
+            with this temperature (in Kelvins).
+        fix_initial_com
+            Whether to fix the center of mass. Only relevant if `initial_temperature`
+            is specified.
+        fix_initial_rot
+            Whether to fix the rotation from the initial velocity distribution. Only
+            relevant if `initial_temperature` is specified.
+        rng_seed
+            Seed for any random number generators.
+
+        Returns
+        -------
+        MolecularDymamics
+            The ASE MolecularDynamics object.
+        """
+
+        # Set defaults
+        dynamics_kwargs = dynamics_kwargs or {}
+        settings = get_settings()
+        dynamics_kwargs["timestep"] = timestep
+        dynamics_kwargs["logfile"] = "-" if settings.DEBUG else self.tmpdir / "md.log"
+        dynamics_kwargs = self._fix_deprecated_md_params(dynamics_kwargs)
+        dynamics_kwargs = convert_md_units(dynamics_kwargs)
+
+        if initial_temperature is not None:
+            MaxwellBoltzmannDistribution(
+                self.atoms,
+                temperature_K=initial_temperature,
+                rng=np.random.default_rng(seed=rng_seed) if rng_seed else None,
+            )
+            if fix_initial_com:
+                Stationary(self.atoms)
+            if fix_initial_rot:
+                ZeroRotation(self.atoms)
```

Comment:
These `Stationary` and `ZeroRotation` classes are _only_ supposed to be used if a `MaxwellBoltzmannDistribution()` is applied, right? That's because the MD classes have their own `fixcm` keyword argument? [I did not see a similar `fixrot` keyword argument, for what it's worth].